### PR TITLE
TICK-0003 created Category DTO.

### PR DESCRIPTION
**Pull request by DanMedl**
created Product DTO.

### CODE REVIEW

File: com.atb/src/main/java/com/atb/Category.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.atb;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/Category.java:1]: move to **com.atb.dtos**

File: com.atb/src/main/java/com/atb/Category.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+import java.util.List;
+
+public class Category {
+    private String code;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/Category.java:6]: add empty line

File: com.atb/src/main/java/com/atb/Category.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    private String description;
+    private Category parentCategory;
+    private List<Category> subCategory;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/Category.java:9]: Please, change name to subCategories (update getters/setters as well)

File: com.atb/src/main/java/com/atb/Category.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    private List<Category> products;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/Category.java:10]: Please, change to List<Product>

File: com.atb/src/main/java/com/atb/Category.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    public Category(String code, String description, Category parentCategory, List<Category> subCategory, List<Category> products) {
+        this.code = code;
```
Comment (Shaff3004) [inline com.atb/src/main/java/com/atb/Category.java:13]: Change constructor to default one

File: com.atb/src/main/java/com/atb/Category.java (hunk 1 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/Category.java:1]: done

File: com.atb/src/main/java/com/atb/Category.java (hunk 2 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/Category.java:6]: done

File: com.atb/src/main/java/com/atb/Category.java (hunk 3 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/Category.java:9]: done

File: com.atb/src/main/java/com/atb/Category.java (hunk 4 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/Category.java:10]: done

File: com.atb/src/main/java/com/atb/Category.java (hunk 5 above)
Comment (DanMedl) [reply com.atb/src/main/java/com/atb/Category.java:13]: done

**Pull request merged**